.github/workflows/build.yml

```diff
@@ -9,44 +9,14 @@ on:
     branches:
       - '**'
 jobs:
-  build_x86_64_legacy_nodejs:
-    name: Build x86_64 (legacy Node.js)
-    runs-on: ${{ matrix.os }}
-    strategy:
-      fail-fast: false
-      matrix:
-        os: [macos-13, ubuntu-22.04, ubuntu-24.04, windows-2019]
-        node: [12, 13, 14, 15, 16, 17, 19]
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v4
-        with:
-          submodules: true
-
-      - name: Install Node v${{ matrix.node }}
-        uses: actions/setup-node@v3
-        with:
-          node-version: ${{ matrix.node }}
-
-      - name: Setup python
-        uses: actions/setup-python@v4
-        with:
-          python-version: '3.8'
-
-      - name: Install dependencies
-        run: npm install --build-from-source
-
-      - name: Package prebuild
-        run: npm run build
-
   build_x86_64:
     name: Build x86_64
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         os: [macos-13, ubuntu-22.04, ubuntu-24.04, windows-2022]
-        node: [18, 20, 21, 22]
+        node: [18, 20, 22]
```

Comment:
Let's throw in 23 and 24 too please

Reply:
Once 24 was released, there's no reason to test also 23, which is a prerelease version of 24.

Reply:
Added